dowhy/timeseries/timeseries.py

```diff
@@ -0,0 +1,36 @@
+import networkx as nx
+import pandas as pd
+from dowhy.utils.plotting import plot
+from typing import List, Tuple
+
+def display_networkx_graph(graph:nx.DiGraph):
+    # Draw and display the graph
+    plot(graph)
+
+def find_lagged_parent_nodes(graph:nx.DiGraph, node:str) -> Tuple[List[str], List[int]]:
```

Comment:
each "significant" function needs to have a docstring. Brief description of what it does, the params it takes, and the output. see other dowhy modules for examples